default/AI/ColonisationAI.py

```diff
@@ -980,13 +993,16 @@ def evaluate_planet(planet_id, mission_type, spec_name, empire, detail=None):
         if have_existing_presence:
             detail.append("preexisting system colony")
             retval *= 1.5
+        supplyval = 0
         if sys_supply < 0:
             if sys_supply + planet_supply >= 0:
-                retval += 30 * (planet_supply - max(-3, sys_supply))
+                supplyval += 30 * (planet_supply - max(-3, sys_supply))
             else:
                 retval += 30 * (planet_supply + sys_supply)  # a penalty
         elif planet_supply > sys_supply and (sys_supply < 2):  # TODO: check min neighbor supply
-            retval += 25 * (planet_supply - sys_supply)
+            supplyval += 25 * (planet_supply - sys_supply)
+        detail.append("sys_supply: %d, planet_supply: %d, supply_val: %.0f" % (sys_supply, planet_supply, supplyval))
```

Comment:
Hmm, right, I think line 655 is off as well-- the ```list(detail)``` should probably just be ```detail``` ; it's been a fair while since I actually printed out any of that detail, though I do try to keep the detail collection up to date since it can be very helpful and would be a big headache to recreate when needed.